### ISSUE TITLE
@aws-amplify/ui-vue causes IE 11 white screen

### ISSUE THREAD

**Issue by tomaseti** [aws-amplify/amplify-js#6926]
**Bug Description**
Framework: Vue 2.6.11
Amplify: 3.3.3
Vue UI: 0.2.22
Browser: IE 11
IE Error Message: SCRIPT1002: Syntax error

Adding the statement import "@aws-amplify/ui-vue" causes a white screen in Internet Explorer 11.  The error messages is SCRIPT1002: Syntax error.  If I remove the import statement the screen renders normally.

**Fix**
To try to fix the issue I installed the @babel/polyfill and added the two import statements at the top of the Main.js file (see below).

**Main.js**
import "core-js/stable";
import "regenerator-runtime/runtime";

import Vue from "vue";
import App from "./App.vue";
import Amplify from "aws-amplify";
import "@aws-amplify/ui-vue";
import aws_exports from "./aws-exports";

Amplify.configure(aws_exports);

Vue.config.productionTip = false;

new Vue({
  render: (h) => h(App),
}).$mount("#app");

**Comment by tomaseti** [comment]
@sammartinez  Thinking about this: I wonder if @aws-amplify/ui-vue should even support such an out of date browser like IE11.  I found a way around it and am wondering if it's worth the time and effort.

**Comment by sammartinez** [comment]
@tomaseti Apologizes on the late response, are you adding polyfills for IE11?

**Comment by tomaseti** [comment]
@sammartinez Yes that's right I added it to make it render in IE11.  However I got around the issue by not importing the whole ui-vue library but just the individual bits that I needed.  So it's not an issue for me now.  Almost feel a bit guilty about asking for help about a crappy browser that really shouldn't be supported.  Unfortunately seems like a few people out there are still using it, in the UK at least.  I'm happy if you want to close this issue down.
